sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2508,6 +2508,14 @@ object functions {
   def current_date(): Column = withExpr { CurrentDate() }
 
   /**
+   * Returns the current date in the given timezone as a date column.
```

Comment:
`@return`?

Reply:
this is consistent with all the other functions in this file.